### PR TITLE
test(scripts/install.ps1): check file ownership and permissions

### PR DESCRIPTION
**Pull request by swiatekm-sumo**
Check file ownership and permissions in Windows install script tests.

### CODE REVIEW

File: pkg/scripts_test/consts_windows.go [hunk 1]
```diff
@@ -23,10 +24,10 @@ const (
...
-	configPathFilePermissions       uint32 = 0440
-	confDPathFilePermissions        uint32 = 0644
-	etcPathPermissions              uint32 = 0551
-	opampDPermissions               uint32 = 0750
+	commonConfigPathFilePermissions uint32 = 0777
+	configPathDirPermissions        uint32 = 0777
+	configPathFilePermissions       uint32 = 0666
+	confDPathFilePermissions        uint32 = 0777
+	etcPathPermissions              uint32 = 0777
+	opampDPermissions               uint32 = 0777
```
Comment (echlebek) [inline pkg/scripts_test/consts_windows.go:32]: While I'm sure there is a good reason for it, this appears to be quite liberal permissions, maybe you could add a comment explaining why?

**Review by echlebek — COMMENTED** [review]
Looks good to me, but I find the 777 and 666 permissions a bit suspicious and could use an explainer.

File: pkg/scripts_test/consts_windows.go (hunk 1 above)
Comment (swiatekm-sumo) [reply pkg/scripts_test/consts_windows.go:32]: Will do. To be honest, I should really use Windows ACLs here and make more detailed assertions. Using os.Stat like we do for Windows or Unix, we can only really get 0444 for readonly files and 0666 for writable ones (+0111 for directories). So currently all this checks is whether the files are writable by their owner (which is the LocalSystem user who owns files for shared Programs.)

**Review by astencel-sumo — APPROVED** [review]

File: pkg/scripts_test/consts_windows.go (hunk 1 above)
Comment (swiatekm-sumo) [reply pkg/scripts_test/consts_windows.go:31]: I added a comment explaining these. Looks like Windows ACLs aren't trivial to check, so I'm going to merge this PR, and add those in a followup.

**Pull request merged**